config/settings.yaml.example

```diff
@@ -54,3 +54,7 @@
 #    :enabled: false
 #  :templates:
 #    :enabled: true
+#  :action:
+#    :enabled: true
```

Comment:
I can keep the old name 'foreman-tasks/action' if you want to, does that help? Otherwise I can call it 'job-processing' if it's any better